packages/flutter/lib/src/material/dialog_theme.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import 'dialog.dart';
+
+/// Defines a theme for [Dialog] widgets.
+///
```

Comment:
Should explain what it means when shape is null

Reply:
Done.